src/components/HeaderWithCloseButton.js

```diff
@@ -138,26 +161,37 @@ class HeaderWithCloseButton extends Component {
                     styles.overflowHidden,
                 ]}
                 >
-                    {this.props.shouldShowBackButton && (
-                        <Tooltip text={this.props.translate('common.back')}>
-                            <Pressable
-                                onPress={() => {
-                                    if (this.props.isKeyboardShown) {
-                                        Keyboard.dismiss();
-                                    }
-                                    this.props.onBackButtonPress();
-                                }}
-                                style={[styles.touchableButtonImage]}
-                            >
-                                <Icon src={Expensicons.BackArrow} />
-                            </Pressable>
-                        </Tooltip>
+                    <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                        {this.props.shouldShowBackButton && (
+                            <Tooltip text={this.props.translate('common.back')}>
+                                <Pressable
+                                    onPress={() => {
+                                        if (this.props.isKeyboardShown) {
+                                            Keyboard.dismiss();
+                                        }
+                                        this.props.onBackButtonPress();
+                                    }}
+                                    style={[styles.touchableButtonImage]}
+                                >
+                                    <Icon src={Expensicons.BackArrow} />
+                                </Pressable>
+                            </Tooltip>
+                        )}
+                        {this.props.shouldShowAvatarWithDisplay && (
+                            <AvatarWithDisplayName
```

Comment:
In the IOU case, we should display the avatar and details of the payer here

Reply:
I believe we do, right? Or maybe can you show me how we're not showing the details of the payer?
https://github.com/Expensify/App/blob/b31c2d136ba3292d0f96c6e7f641539dcf103736/src/libs/ReportUtils.js#L723-L729C6

Reply:
Ah yea, my bad!